Add JIRA workflow doc (#3)
folio-org/tech-council: TECH_EVAL_JIRA.md

> # Technical Evaluation Process JIRA Workflow

Reviewer: Might we want to make it explicit that this is a _module_ technical evaluation, or is `technical evaluation` a shorthand for this?

## Overview
This document outlines the JIRA Project and Workflow used by the Technical Evaluation Process.

### JIRA Project information
* The project where Technical Evaluation submissions should be created is called [Technical Council Requests](https://issues.folio.org/projects/TCR).
* The only available issue type and one that should be used for submissions is "Task".

## JIRA Workflow
![Techical Council Requests Workflow](assets/workflow.png)

**SUBMITTED** Initial status for newly created submissions.

**UNDER EVALUATION** Once the sumbmission is reviewed by the TC and an evaluation group is formed, the TC indicates this by transitioning the issue to this state.

> **CANCELLED** If the submitter decides to withdraw the submission (alone or together with the evalation group), he/she transitions the issue to this state.

Reviewer: ```suggestion
**CANCELLED** If the submitter decides to withdraw the submission (alone or together with the evalation group), they transition the issue to this state.
```
